packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/BareManifest.kt

```diff
@@ -19,7 +19,6 @@ class BareManifest private constructor(
   private val mScopeKey: String,
   private val mCommitTime: Date,
   private val mRuntimeVersion: String,
-  private val mMetadata: JSONObject?,
```

Comment:
Since we do want to copy the metadata over eventually, what do you think about reverting these changes?

Reply:
I don't think we ever want to store just the manifest's (theoretical) `metadata` field in the database separately from the raw manifest, which is already its own separate field in this class (and constructor). Maybe I'm misunderstanding what you're asking though? 🤔

Reply:
Okay, I see. I was thinking we might want `BareManifest.metadata` to be defined or for there to be a way to get the metadata directly from an UpdateEntity. But it looks like we don't need either of those.

Reply:
Gotcha. Yeah,